conda_forge_tick/migrators/core.py

```diff
@@ -486,16 +486,11 @@ def migrate(
         """
         return self.migrator_uid(attrs)
 
-    def pr_body(
-        self, feedstock_ctx: ClonedFeedstockContext, add_label_text=True
-    ) -> str:
-        """Create a PR message body
-
-        Returns
-        -------
-        body: str
-            The body of the PR message
-            :param feedstock_ctx:
+    @staticmethod
```

Comment:
Because the signature of `pr_body` in the `Migrator` class does not have an `add_label_text` parameter. Children cannot add parameters to the methods they inherit.

Reply:
Because all the children of `Migrator` did not add `add_label_text` to the signature of `pr_body` although they have to. Like this, it makes more sense inheritance-wise IMO.
(Children would ignore the parameter anyway.)